tests/components/accuweather/__init__.py

```diff
@@ -1 +1,45 @@
 """Tests for AccuWeather."""
+import json
+
+from homeassistant.components.accuweather.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry, load_fixture
+
+
+async def init_integration(
+    hass, forecast=False, unsupported_icon=False
+) -> MockConfigEntry:
+    """Set up the AccuWeather integration in Home Assistant."""
+    options = {}
+    if forecast:
+        options["forecast"] = True
+
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        title="Home",
+        unique_id="0123456",
+        data={
+            "api_key": "32-character-string-1234567890qw",
+            "latitude": 55.55,
+            "longitude": 122.12,
+            "name": "Home",
+        },
+        options=options,
+    )
+
+    location = json.loads(load_fixture("accuweather/location_data.json"))
+    current = json.loads(load_fixture("accuweather/current_conditions_data.json"))
+    forecast = json.loads(load_fixture("accuweather/forecast_data.json"))
+
+    if unsupported_icon:
+        current["WeatherIcon"] = 999
+
+    with patch("accuweather.AccuWeather._async_get_data", return_value=location), patch(
```

Comment:
Why are we patching `_async_get_data`? It will never be called since we're patching the other two api methods.

Reply:
You're right, it isn't needed here, I'm blind 🤦